templates/docker-compose.mustache

```diff
@@ -9,6 +9,9 @@ services:
     {{#image}}
     image: {{name}}
     {{/image}}
+    {{#hostname}}
```

Comment:
In adding `site-docker-compose-utils`, along with that, add this default mustache template inside core. And make according use changes in `site-command` and here.